tests/test-team/pages/template-mgmt-base-page.ts

```diff
@@ -34,7 +34,7 @@ export class TemplateMgmtBasePage {
   }
 
   async navigateTo(url: string) {
-    await this.page.goto(url, { waitUntil: 'load' });
+    await this.page.goto(url, { waitUntil: 'commit' });
```

Comment:
what does this change do?

Reply:
"load" is default and wanted to try a different option as to when the operation is considered complete by playwright as ive read that it can a positive effect on test speed ... it didnt do anything and i forgot to change it back.

Reply:
removed